JJG/Ping.php

```diff
@@ -27,37 +27,41 @@ class Ping {
 
   private $host;
   private $ttl;
+  private $path;
   private $port = 80;
   private $data = 'Ping';
 
-  /**
-   * Called when the Ping object is created.
-   *
-   * @param string $host
-   *   The host to be pinged.
-   * @param int $ttl
-   *   Time-to-live (TTL) (You may get a 'Time to live exceeded' error if this
-   *   value is set too low. The TTL value indicates the scope or range in which
-   *   a packet may be forwarded. By convention:
-   *     - 0 = same host
-   *     - 1 = same subnet
-   *     - 32 = same site
-   *     - 64 = same region
-   *     - 128 = same continent
-   *     - 255 = unrestricted
-   *   The TTL is also used as a general 'timeout' value for fsockopen(), so if
-   *   you are using that method, you might want to set a default of 5-10 sec to
-   *   avoid blocking network connections.
-   *
-   * @throws \Exception if the host is not set.
-   */
-  public function __construct($host, $ttl = 255) {
+    /**
+     * Called when the Ping object is created.
+     *
+     * @param string $host
+     *   The host to be pinged.
+     * @param int $ttl
+     *   Time-to-live (TTL) (You may get a 'Time to live exceeded' error if this
+     *   value is set too low. The TTL value indicates the scope or range in which
+     *   a packet may be forwarded. By convention:
+     *     - 0 = same host
+     *     - 1 = same subnet
+     *     - 32 = same site
+     *     - 64 = same region
+     *     - 128 = same continent
+     *     - 255 = unrestricted
+     *   The TTL is also used as a general 'timeout' value for fsockopen(), so if
+     *   you are using that method, you might want to set a default of 5-10 sec to
+     *   avoid blocking network connections.
+     * @param string $path
+     *   Full path to your ping command. (e.g. /sbin/ping on OSX or /bin/ping on Linux)
```

Comment:
Can you just leave one path example, like `e.g. /bin/ping on Linux`? That keeps the line < 80 cols and still gets the point across.